src/components/panel/_panel.scss

```diff
@@ -80,6 +80,24 @@
     }
   }
 
+  &--warn-ondark {
```

Comment:
Can this not be styled in the same way the footer warning is? I think we will be able to reuse more code if it can

Reply:
Let me take a look at how that is is being used Rich.